### ISSUE TITLE
Поиск по Кастом филд Товара

### ISSUE THREAD

**Issue by AlexanderKopyl** [amocrm/amocrm-api-php#154]
Как можно найти елемент в Товарах по Кастом полю ?

**Comment by bessudnov** [comment]
В данный момент в библиотеке нет такого функционала
В бэклоге записана задача

**Comment by andreybolonin** [comment]
@bessudnov а в `customers` поиск по customFields работает?

**Comment by bessudnov** [comment]
> @bessudnov а в `customers` поиск по `customFields` работает?

да, если аккаунт подключен к альфа-фильтрации \AmoCRM\Filters\CustomersFilter::setCustomFieldsValues
